pallets/handles/src/utils/confusables_generator.rs

```diff
@@ -0,0 +1,51 @@
+use std::{
+	fs::File,
+	io::{BufRead, BufReader, Write},
+};
+
+// Helper function to generate a BTreeMap which maps each confusable character to its
+// canonical counterpart for each line in `confusable_characters.txt`.
+//
+// The first character of each line represent the canonical character (value) in the map that each
+// subsequent character of the line (key) maps to.
+//
+// Usage:
+// `rustc utils.rs`
+// `./utils`
+//
+// outputs to confusables.rs
+fn convert_confuseables_to_unicode_escaped() {
+	let input_file = File::open("confusable_characters.txt");
+	assert!(input_file.is_ok());
+
+	let mut output_file = std::fs::File::create("confusables.rs").expect("create failed");
+
+	let reader = BufReader::new(input_file.ok().unwrap());
+
+	_ = output_file.write_all(b"// ******************************************************\n");
+	_ = output_file.write_all(b"// ***** THIS FILE IS AUTO-GENERATED.  DO NOT EDIT! *****\n");
```

Comment:
Suggest the header be more like weights, where it includes how it's generated.

Reply:
This is going to be replaced with phf_codegen.